src/Pact/Typechecker.hs

```diff
@@ -1236,10 +1236,10 @@ typecheckTopLevel (Direct d) = die (_tInfo d) $ "Unexpected direct ref: " ++ abb
 
 -- | Typecheck all productions in a module.
 typecheckModule :: Bool -> DynEnv -> ModuleData Ref -> IO ([TopLevel Node],[Failure])
-typecheckModule dbg dynEnv (ModuleData (MDModule m) rm) = do
+typecheckModule dbg dynEnv (ModuleData (MDModule m) rm _) = do
```

Comment:
It's alright to ignore checking transitive dependencies here, as they're all re-inlined for this function.